backend/config/urls.py

```diff
@@ -28,6 +28,10 @@
     url(r"^api/auth/jwt/create_token/", obtain_jwt_token),
     url(r"^api/auth/jwt/refresh_token/", refresh_jwt_token),
     url(r"^api/auth/jwt/verify_token/", verify_jwt_token),
+    url(r"^api/meta-test",

```

Comment:
Are both of these lines required here?

Reply:
yes, I've tried without one or the other, and can't access the endpoint without both